src/components/common/CreateNewLibraryDialog.tsx

```diff
@@ -142,6 +142,19 @@ const CreateNewLibraryDialog: React.FC<TestProps> = ({
     setFocusedField(field);
   };
 
+  const getFormikErrorMessage = (() => {
+    if (
+      (formik.touched["cqlLibraryName"] || focusedField === "cqlLibraryName") &&
+      formikErrorHandler("cqlLibraryName")
+    ) {
+      return formikErrorHandler("cqlLibraryName");
+    } else {
+      if (formik.values.model === Model.QDM_5_6) {
+        return "Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters except of underscore for QDM.";
```

Comment:
"except of underscore for QDM." reads a little funny. Maybe just "except underscore"?

Reply:
updated